src/pyhf/pdf.py

```diff
@@ -302,6 +313,9 @@ def _precompute(self):
         tensorlib, _ = get_backend()
         self.nominal_rates = tensorlib.astensor(self._nominal_rates)
 
+    def has_pdf(self):
```

Comment:
Needs a docstring, but I can add this after we're done with all the benchmark testing so that I don't trigger unnecessarily long builds.

Reply:
whops missed this

Reply:
done